libs/python.json

```diff
@@ -145,6 +148,7 @@
   {"id":"py.pymysql","imports":["pymysql"],"name":"PyMySQL","repo":"PyMySQL/PyMySQL","tags":["mysql"],"tech":["storage"],"status":"awaiting-model"},
   {"id":"py.redis","imports":["redis"],"name":"redis","repo":"andymccurdy/redis-py","tags":["nosql"],"tech":["storage"],"status":"model-ready"},
   {"id":"py.sqlalchemy","imports":["sqlalchemy"],"name":"sqlalchemy","repo":"zzzeek/sqlalchemy","tags":["sql"],"tech":["storage"],"status":"model-ready"},
+  {"id":"py.sqlparse","imports":["sqlparse"],"name":"sqlparse","repo":"andialbrecht/sqlparse","tags":["sql-parser"],"tech":["storage"],"status":"awaiting-model"},
```

Comment:
tag -> sql, tech: parsing might be a better fit, let me know if you agree

Reply:
yeah, tech should be parsing